components/RecipeDetail.vue

```diff
@@ -99,6 +99,7 @@ export default {
   },
   computed: {
     serverBaseUrl: () => process.env.serverBaseUrl,
+    serverImagesUrl: () => process.env.serverImagesUrl,
```

Comment:
maybe `serverFilesUrl`? I guess this url could be use for other file types than images